src/reconcile/rec_write.c

```diff
@@ -2671,8 +2671,6 @@ __rec_copy_prev_addr(WT_SESSION_IMPL *session, WTI_RECONCILE *r)
               mod->mod_replace.block_cookie_size, &multi->addr.block_cookie));
             multi->addr.block_cookie_size = mod->mod_replace.block_cookie_size;
             multi->addr.type = mod->mod_replace.type;
-            __wt_free(session, mod->mod_replace.block_cookie);
```

Comment:
We already copied this address into the `multi->addr.block_cookie` a couple of lines above. May be we need to use the above pointer somewhere we are accessing?